packages/core/content-type-builder/admin/src/components/FormModal/SubHeader/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Typography } from '@strapi/design-system/Typography';
+import { useCustomFields } from '@strapi/helper-plugin';
+import upperFirst from 'lodash/upperFirst';
+import { getTrad } from '../../../utils';
+import { getModalTitleSubHeader } from '../utils';
+
+const SubHeader = ({
+  actionType,
+  modalType,
+  forTarget,
+  kind,
+  step,
+  attributeType,
+  attributeName,
+  customFieldUid,
+}) => {
+  const { formatMessage } = useIntl();
+  const customFielsRegistry = useCustomFields();
+
+  if (customFieldUid) {
+    const { intlLabel } = customFielsRegistry.get(customFieldUid);
```

Comment:
Oh yeah that's right! Actually I think it won't work on my PR because the forms are not in a React component. Let's discuss this part on discord this afternoon